message tab_SkillLastTimeTable 
{
	optional string tname = 1;
	repeated tab_SkillLastTime tlist = 2;
}

message tab_SkillLastTime
{
	optional int32 id = 1;
	optional string Comment = 2;
	optional int32 LastTime = 3;
}